docs/README.md

```diff
@@ -87,14 +92,105 @@ The simulation will then try to connect with the ROS2 driver every few seconds.
 
 ## Connecting with ROS2
 
-To launch with RobotStudio, set `use_fake_hardware:=false`. As far as ROS is aware, RobotStudio is a real robot:
+The driver reqiures some network information to connect to EGM and RWS. This information is store in the ros2_control [robot description file](../robot_specific_config/abb_irb1200_support/urdf/irb1200.ros2_control.xacro). Change the following lines as required:
+
+```
+<param name="rws_port">80</param>
+<param name="rws_ip">127.0.0.1</param>
+<param name="rob1egm_port">6511</param>
+```
+
+- `rws_port` should be the port for RWS
+- `rws_ip` should be the IP address of the RobotStudio computer (see [Adding the local computer to the virtual controller whitelist](#adding-the-local-computer-to-the-virtual-controller-whitelist))
+- `<MECHANICAL_UNIT_GROUP_NAME>egm_port` should match the `Remote port` setting from the RobotStudio controller configuration
+     - Change `<MECHANICAL_UNIT_GROUP_NAME>` to match the mechanical unit group name - in the sample Pack and Go file, this is `rob1`
+     - If not using MultiMove, change the paramter name to be `egm_port`
+
+To launch with RobotStudio, set `use_fake_hardware:=false` and `robotstudio_ip:=<ROBOTSTUDIO_IP>`, substituting `<ROBOTSTUDIO_IP>` with the IP of the RobotStudio computer. As far as ROS is aware, RobotStudio is a real robot:
 
-    ros2 launch abb_bringup abb_control.launch.py description_package:=abb_irb1200_support description_file:=irb1200_5_90.xacro launch_rviz:=false moveit_config_package:=abb_irb1200_5_90_moveit_config
+    ros2 launch abb_bringup abb_control.launch.py description_package:=abb_irb1200_support description_file:=irb1200_5_90.xacro launch_rviz:=false moveit_config_package:=abb_irb1200_5_90_moveit_config use_fake_hardware:=false robotstudio_ip:=<ROBOTSTUDIO_IP>
 
 After launching the controllers, launch MoveIt:
 
     ros2 launch abb_bringup abb_moveit.launch.py robot_xacro_file:=irb1200_5_90.xacro support_package:=abb_irb1200_support
 
+## Connecting to RWS
```

Comment:
IS RWS 100% mandatory now, or only for external axes? I don't like how it makes the README more complicated.

Reply:
This PR currently makes RWS mandatory for all cases - RWS is used to get the robot setup from RobotStudio. We might be able to hardcode the correct configuration for a single robot with no external axis if necessary, but if the issue is with README length, connecting to RWS with the default settings shouldn't be very complicated.

A lot of the README is extra information on how to change configurations that are not strictly necessary and can be removed... I'm of the opinion that it's nice to have all that relevant information grouped together somewhere, but it doesn't necessarily have to be here.